e2e/basic.spec.ts

```diff
@@ -17,20 +18,27 @@ const templateFrameworks: TemplateFramework[] = [
   "express",
   "fastapi",
 ];
-const dataSources: string[] = ["--no-files", "--example-file"];
+const dataSources: string[] = ["--no-files", "--llamacloud"];
 const templateUIs: TemplateUI[] = ["shadcn", "html"];
 const templatePostInstallActions: TemplatePostInstallAction[] = [
   "none",
   "runApp",
 ];
 
+const llamaCloudProjectName = "create-llama";
+const llamaCloudIndexName = randomUUID();
```

Comment:
@thucpn can we use a fixed name like `e2e-test` otherwise we need to remove the index after test run for clean up